src/keyword.rs

```diff
@@ -60,58 +56,26 @@ impl Keyword {
         struct NewKeyword<'a> {
             keyword: &'a str,
         }
-        sql_function!(lower, lower_t, (x: ::diesel::types::Text) -> ::diesel::types::Text);
```

Comment:
I tend to keep these definitions close to where they're used for functions which can have varying signatures, but I don't feel strongly about it.